src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -475,14 +478,15 @@ def test_aks_create_default_service_with_virtual_node_addon(self, resource_group
             'location': resource_group_location,
             'service_principal': _process_sp_name(sp_name),
             'client_secret': sp_password,
-            'resource_type': 'Microsoft.ContainerService/ManagedClusters'
+            'resource_type': 'Microsoft.ContainerService/ManagedClusters',
+            'subnet_id': subnet_id
         })
 
         # create cluster with virtual-node addon
         create_cmd = 'aks create --resource-group={resource_group} --name={name} --location={location} ' \
                      '--dns-name-prefix={dns_name_prefix} --node-count=1 --ssh-key-value={ssh_key_value} ' \
                      '--service-principal={service_principal} --client-secret={client_secret} --enable-addons virtual-node ' \
-                     '--aci-subnet-name foo --vnet-subnet-id bar'
+                     '--aci-subnet-name foo --vnet-subnet-id "{subnet_id}"'
```

Comment:
There is a validator to check that `--vnet-subnet-id` should be a resource id.

This test still fails with `Operation failed with status: 'Bad Request'. Details: ACI Connector requires azure network plugin`, leave that to AKS team to fix.